app/decorators/project_decorator.rb

```diff
@@ -0,0 +1,18 @@
+class ProjectDecorator < Draper::Decorator
+	delegate_all
+
+	def descriptive_follow_text
+		if object.follows.count == 0
+			""
+		elsif object.follows.count == 1
```

Comment:
you can make the following 2 branches into a single `else` with `else count = object.follows.count  " | #{count} #{pluralize(count, 'follow')}"`